src/main/resources/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionWrapper/index.jelly

```diff
@@ -0,0 +1,11 @@
+<?jelly escape-by-default='true'?>
+<st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+    <l:layout title="${it.displayName}" norefresh="true">
+        <st:include it="${it.parent}" page="sidepanel.jelly"/>
+        <l:main-panel>
+            <j:forEach var="exwsAction" items="${it.allActions}">
+                <st:include it="${exwsAction}" page="index.jelly"/>
+            </j:forEach>
+        </l:main-panel>
+    </l:layout>
+</st:compress>
```

Comment:
BTW I doubt that st:compress is really required here. The page is pretty small. Not important